### ISSUE TITLE
Per thread view

### ISSUE THREAD

**Issue by zzarm** [Compaile/ctrack#8]
it would be cool to be able to support a per-thread timeline view (e.g. time intervals annotated by the functions being called by each thread)

**Comment by Compaile** [comment]
Can you explain more about your idea? Do you think about a sort of flame-graph ?

**Comment by zzarm** [comment]
Yes, I was thinking of something exactly like a flame graph, with time on the horizontal axis and activities from each thread stacked vertically, each occupying individual lines and extending along the time axis. Different activities from each thread can be represented as intervals with start and end timestamps as well as the the name of function being tracked.  Currently, the stats show the total time of the tracked functions run by all the threads. Even though the overlapping time is excluded, if the threads do not start calling the function at the same time, I feel that the resulting times for the functions are not easy to analyze.
